application/src/js/all/__setup_ajax_csrf.js

```diff
@@ -0,0 +1,5 @@
+$.ajaxSetup({
```

Comment:
Should this be in the `src/js/cms/*` folder, as we don't need it on the static site?